lib/provider/camunda/parts/VariableMappingProps.js

```diff
@@ -313,13 +318,13 @@ module.exports = function(group, element, bpmnFactory, translate) {
       if (mapping) {
         if (!values.source) {
           validation.source =
-            'Mapping must have a ' + (values.sourceLabel.toLowerCase() || 'value');
+            translate('Mapping must have a') + ' ' + (values.sourceLabel.toLowerCase() || 'value');
         }
 
         var inOutType = getInOutType(mapping);
 
         if (WHITESPACE_REGEX.test(values.source) && inOutType !== 'sourceExpression') {
-          validation.source = values.sourceLabel + ' must not contain whitespace';
+          validation.source = values.sourceLabel + ' ' + translate('must not contain whitespace');
```

Comment:
```suggestion
          validation.source = translate('{label} must not contain whitespace', { label: values.sourceLabel });
```
Please use template replacements, as the variable may not be allowed at first position in a localized version. See https://github.com/bpmn-io/diagram-js/blob/master/lib/i18n/translate/translate.js for reference.

Reply:
Thanks!